module-2/backend/week_three.md

```diff
@@ -10,16 +10,38 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What are the 3 main features in an ERD?
+  * Entities (objects), Relationships (verbs), Functional Dependency (cardinality)
 2. Create an ERD for the following database: Restaurants, Customers, Items, Ingredients, Beverages, Orders, Vendors.
+  * Did drawing
 3. What is the entry at the command line to create a new rails app?
+  * rails new
 4. What do Models generally inherit from in rails?
+  * ActiveRecord::Base
 5. What do Controllers generally inherit from in a rails project?
+  * ApplicationController
 6. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+  * get '/horses/:id' 'horses#show'
+  * resources :horses, only: [:show] 
```

Comment:
i like that you have both options 👍 

the first one needs a hashrocket though to work...

```
get '/horses/:id' => 'horses#show'
```